build-tools/scripts/PrepareWindows.targets

```diff
@@ -5,7 +5,9 @@
     <_TopDir>$(MSBuildThisFileDirectory)..\..</_TopDir>
     <_XAPrepareExe>$(MSBuildThisFileDirectory)..\xaprepare\xaprepare\bin\$(Configuration)\xaprepare.exe</_XAPrepareExe>
     <_XAPrepareStandardArgs>--no-emoji --run-mode=CI -a</_XAPrepareStandardArgs>
```

Comment:
@grendello I think you mentioned we could put `-a` only if running on CI, is that correct?

If so, we should fix that here.

Reply:
I've moved all of the params behind the new CI flag.